packages/portfolio-contract/src/portfolio.contract.ts

```diff
@@ -236,12 +236,17 @@ export const contract = async (
       assert.fail('use off-chain queries');
     },
   };
+  const portfoliosNode = E(storageNode).makeChildNode('portfolios');
 
   const resolverZone = zone.subZone('CCTPResolver');
   const {
     client: resolverClient,
     invitationMakers: makeResolverInvitationMakers,
-  } = prepareResolverKit(resolverZone, zcf, vowTools)();
+  } = prepareResolverKit(resolverZone, zcf, {
```

Comment:
we need to use `makeOnce()` here. I guess it's not critical to this week's goal, so let's postpone it...

 - #11786